README.md

```diff
@@ -1,5 +1,4 @@
-Bazecor
-=======
+# Bazecor
```

Comment:
I changed this line to make the markdown header syntax consistent,
so that tools like [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) is more happy about it.

Reply:
Thanks for the helpout @unional ! 😄